metadetect/lsst/tests/test_lsst_metadetect.py

```diff
@@ -395,6 +442,7 @@ def test_lsst_metadetect_mfrac_ormask(show=False):
 
 
 if __name__ == '__main__':
-    # test_lsst_zero_weights(show=True)
+    # test_lsst_metadetect_am()
+    test_lsst_masked_as_bright(show=True)
     # test_lsst_metadetect_smoke('wmom', 'False')
-    test_lsst_metadetect_mfrac_ormask(show=True)
+    # test_lsst_metadetect_mfrac_ormask(show=True)
```

Comment:
It'd likely be better to remove this block and then when you are working locally use syntax like `pytest -k <name of test>` to select specific tests. Otherwise we get churn in the diff here that is meaningless.

Reply:
You can't bring up plots when running pytest